build/common.mk

```diff
@@ -105,9 +105,9 @@ endif
 PYTHON ?= python3
 PIP ?= pip3
 
-PULUMI_BIN          := $(PULUMI_ROOT)/bin
-PULUMI_NODE_MODULES := $(PULUMI_ROOT)/node_modules
-PULUMI_NUGET        := $(PULUMI_ROOT)/nuget
+PULUMI_BIN          ?= $(PULUMI_ROOT)/bin
```

Comment:
These should all be created in PULUMI_ROOT which can already be overriden. I don't see the need for the user to be able to set these aswell?

Reply:
Indeed I removed that.